command/sub8_missions/nodes/move_command

```diff
@@ -37,14 +47,53 @@ def main(args):
         fprint("Waiting for odom...")
         yield sub.tx_pose()
         fprint("Odom found!", msg_color='green')
-
+        rospack = rospkg.RosPack()
```

Comment:
This line should go under the teleport branch below. No need to even create a smaller name for something that's only used once though.